Src/IronPython/Runtime/ByteArray.cs

```diff
@@ -14,6 +14,7 @@
 
 using IronPython.Runtime.Operations;
 using IronPython.Runtime.Types;
+using System.Text;
```

Comment:
nit: Move to up to the "using System" group?

Reply:
OK